libsolidity/interface/SourceReferenceFormatter.cpp

```diff
@@ -55,7 +55,7 @@ void SourceReferenceFormatter::printSourceLocation(SourceLocation const* _locati
 		}
 		if (line.length() > 150)
 		{
-			line = " ... " + line.substr(startColumn, locationLength) + " ... ";
+			line = " ... " + line.substr(0, 75) + " ... ";
```

Comment:
Hm, I think this should rather be something like `"..." + line.substr(max(0, startColumn - 35), min(locationLength + 70, line.length())) + " ....";`, plus removing the "..." if nothing was actually cut off.